packages/components/src/components/switch/switch.examples.md

```diff
@@ -44,8 +44,8 @@ interaction. When used with hidden label, it's best practice to provide a descri
 
 ## Alignment + Stretch
 
-The `label` can be aligned to the `right` (default) or to the `left` in addition with enabled `stretch` property which
-is recommended on mobile views.
+The `label` can be aligned to the `end` (default) or to the `start` in addition with enabled `stretch` property which is
```

Comment:
Didn't we usually add a inline notification with the deprecation notice?

Reply:
Good point, I'll add some.